README.md

```diff
@@ -337,5 +326,87 @@ Contact the dev team for the following:
 
 Once you have access to AWS LZ2 platform, login to the any of the environments to get access to the following: 
 
+- Project .env file can be found in the prod parameter store under pcc/.env
 - Database passwords
 - AWS Local Config - Service Account Tokens
+
+
+## Deployments
+
+Command Reference: 
+
+Build backend:
+```bash
+make build-backend
+```
+
+Terraform:
+```bash
+make plan
+```
+
+```bash
+make apply
+```
+Tag Based Deployments:
+```bash
+make tag-<env>
+```
+
+Deployments and builds are initiated via GitHub actions.
+We have four environments, each with a corresponding GHA yaml file:
+
+### PROD
+
+- requires an approver
+- initiate the deployment by tagging a release, ie: git tag 2.0.0 && git push origin 2.0.0
+- first, deploy this tag to dev: run make tag-dev from the command line
+    - verify the build and run any tests that should run on dev
+- next, deploy this tag to test: run make tag-test from the command line
+    - ensure that QA signs off on the release
+- finally, once approved by the Quality Assurance Analyst, you can run: make tag-prod from the command line
+    - this will initiate the production deployment (P.O. sign off is required)
+- reminder: you must run make plan and make apply manually from the command line to apply any infra updates via terraform
+    
+
+### TEST
+
+- this is our staging environment and most closely resembles prod
+- the data from prod is automatically synced into test
+- parsing, reconciliation and report generation are all automated in this environment (as they are on prod)
+- the lambdas in this environment should not be triggered manually
+- testing should be done with the existing data, as it is (vs upload test cases etc)    
+- to deploy to test:
+- first, deploy to dev: run make tag-dev from the command line    
+    - verify the build and run any tests that should run on dev
+    - ensure that you have received approval from QA analyst
+- next, deploy to test: run make tag-test from the command line
```

Comment:
insert a tab, remove "next, deploy to test:" and just keep "run make tag-test from the command line" since we have already stated "to deploy to test:" on line 379